test/test_torch.py

```diff
@@ -6470,6 +6470,12 @@ def test_item(self, device, dtype):
         t = torch.ones((), device=device, dtype=dtype)
         self.assertEqual(1, t.item())
 
+    @onlyCPU
```

Comment:
This feels like an anti pattern? I.e. why it needs to be limited to just CPU? But will fix it myself in the followup PR